src/peergos/shared/user/fs/TagIndexingService.java

```diff
@@ -0,0 +1,95 @@
+package peergos.shared.user.fs;
+
+import peergos.shared.Crypto;
+import peergos.shared.NetworkAccess;
+import peergos.shared.storage.auth.Bat;
+import peergos.shared.user.*;
+import peergos.shared.user.fs.transaction.Transaction;
+import peergos.shared.user.fs.transaction.TransactionService;
```

Comment:
🟢 
Not sure how imports work in Java but are those correct? Couldn't we just write `import peergos.shared.user.*;`?

Reply:
Wildcard-based imports should not affect runtime performance, however it is often not recommended due to clarity with respect to what is actually imported and the possible existence of classes that have the same name in different packages, which would require to specify the package name in the place, where the class is used. So technically we cloud do this, however many projects even prepare special IDE configurations in order to avoid automatic conversions of specific imports to wildcard imports. So this depends on the convention we agree to use :)